raffle.py

```diff
@@ -2,6 +2,7 @@
 import random
 import sys
 from random import shuffle
+totalWinners = 42

```

Comment:
Y'all really killing me with this hardcoded global variable stuff.

This and the csv file should probably be handled by argparse.

Obviously we are on a bit of a time crunch, but we should clean this up for next time.

Reply:
Totally my fault...I just wanted a real quick way to change the winner count day-of if anything changes. It is just a tiny script so I didn't see too much harm in it.

Reply:
Nah, James did it with the argv stuff to begin with so there wasn't a great place for you to add it. It'll be fine for now, but we should refactor eventually.